### ISSUE TITLE
FDR control

### ISSUE THREAD

**Issue by ShadoWyyyy** [smith-chem-wisc/MetaMorpheus#2357]
hello, good software, easy to use until I can not find q value settings, there is no option for setting q value or FDR threshold for final quality control of the results, or did I miss it ? I check every option in the XLSearch task because usually 0.05 is set for cross-linking.

**Comment by trishorts** [comment]
you are correct. for now you have to simply ignore anything in the output with a q-value greater than your desired tolerance.

I will create an issue and try to add this feature to a future release of MetaMorpheus. Please track this issue. 

https://github.com/smith-chem-wisc/MetaMorpheus/issues/2358
